lib/index.js

```diff
@@ -1,6 +1,15 @@
 var Parser = require("./Parser.js");
 var DomHandler = require("domhandler");
+var EVENTS = require('./Events');
 
+// rollup brings in too many dependencies for our use case, here is the minimal exports that work
+
+module.exports = {
+    Parser: Parser,
+    DomHandler: DomHandler,
+    EVENTS: EVENTS
+};
+/* Previous exports...
```

Comment:
This should not be removed.

Reply:
It was done for our use case, reverted now.